config/database.yml

```diff
@@ -22,4 +22,4 @@ test:
 
 production:
   <<: *default
-  database: ontohub
+  database: ontohub_development
```

Comment:
production uses the development database?

Reply:
oops

Reply:
Fixed by commit --amend. I switched this for testing purposes and forgot to switch it back.